src/test/java/com/artipie/npm/NpmProxyITCase.java

```diff
@@ -90,14 +94,14 @@ void installFromProxy() throws Exception {
                 )
             ),
             "npm", "install", NpmProxyITCase.PROJ, "--registry",
-            "http://artipie-proxy:8080/my-npm-proxy"
+            String.format("http://artipie-proxy:%s/%s", port, repo)
         );
         this.containers.assertArtipieContent(
             "artipie-proxy",
             "Package was not cached in proxy",
-            String.format(
-                "/var/artipie/data/my-npm-proxy/%s/-/%s-1.0.1.tgz",
-                NpmProxyITCase.PROJ, NpmProxyITCase.PROJ
+            String.format("/var/artipie/data/%s", repo)
+            .concat(
+                String.format("/%s/-/%s-1.0.1.tgz", NpmProxyITCase.PROJ, NpmProxyITCase.PROJ)
```

Comment:
I'd suggest simplifying this expression by using only `String.format` without any concatenation

Reply:
@olenagerasimova thanks for suggestion!
I have added according fix for it. 
Could you please to review it again?